README.md

```diff
@@ -72,6 +72,34 @@ session
   .catch(function(error) {
     console.log(error);
   });
+
+  //run queries in a transaction
```

Comment:
s/query/statement/g, so that we're consistent throughout on calling these "statements"